.github/workflows/ci.yaml

```diff
@@ -153,6 +153,17 @@ jobs:
         kubectl get nodes
         kubectl get pods -A
 
+    - name: Install cosign
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines +148 to +156]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Q. I think I'm reading this as our `make deploy` will control the deployment config for the chainsaw tests step just below. Does this mean we need different sets of workflows for different sets of chainsaw tests? (for example enabling gateway or bindings features, or changing the behaviour of the ingress features)

Perhaps we could use different `try -> apply -> file` to apply different sets of manifests generated by `helm template`, then cleanup the operator in-between tests?<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/ngrok/ngrok-operator/506?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
There's different solutions for this, in kyverno we use a GH job matrix to deploy kyverno with different configs.

We can meet today if you want and look at that together.

Reply:
Talked with @jonstacks and we'll do a deploy of all features enabled so we can generally test a swath of the expected behaviour for now. Then we can come back with more specific cases if desired.